scripts/lib.sh

```diff
@@ -52,3 +52,26 @@ function get_finality_delay() {
 function sat_to_btc() {
     echo "scale=8; $1/100000000" | bc | awk '{printf "%.8f\n", $0}'
 }
+
+function sent_and_proof_btc() {
+  #$1 = PEG_IN_AMOUNT
+  #$2 = USE_GATEWAY
```

Comment:
Why not just bind variables instead of writing a comment that is more likely to diverge?